### PR TITLE
docs: update docs with fixes for helm, quickstart and install pages

### PR DESCRIPTION
**Pull request by aaron-prindle**

### CODE REVIEW

File: docs-v2/content/en/docs/pipeline-stages/deployers/helm.md [hunk 1]
```diff
@@ -22,14 +22,15 @@ Skaffold supports projects set up to render and/or deploy with Helm, but certain
 Skaffold no longer requires the intricate configuring of `artifactOverrides` or `imageStrategy` fields. See docs [here]({{< relref "#image-reference-strategies" >}}) on how old `artifactOverrides` and `imageStrategy` values translate to `setValues` entires in the latest Skaffold schemas (`apiVersion: skaffold/v3alpha1` or skaffold binary version `v2.0.0` onwards)
 {{< /alert >}}
 
-{{< alert title="In Skaffold `v2` the primary difference between the helm renderer (`manifest.helm.*`) and the helm deployer `deploy.helm.*` is the use of `helm install` vs `helm install`" >}}
+{{< alert title="Note" >}}
+In Skaffold `v2` the primary difference between the helm renderer (`manifest.helm.*`) and the helm deployer (`deploy.helm.*`) is the use of `helm install` vs `helm install
```
Comment (ericzzzzzzz) [inline docs-v2/content/en/docs/pipeline-stages/deployers/helm.md:26]: should this be `helm template` vs `helm install`?

File: docs-v2/content/en/docs/pipeline-stages/renderers/helm.md [hunk 2]
```diff
@@ -22,7 +22,8 @@ Skaffold supports projects set up to render and/or deploy with Helm, but certain
 Skaffold no longer requires the intricate configuring of `artifactOverrides` or `imageStrategy` fields. See docs [here]({{< relref "#image-reference-strategies" >}}) on how old `artifactOverrides` and `imageStrategy` values translate to `setValues` entires in the latest Skaffold schemas (`apiVersion: skaffold/v3alpha1` or skaffold binary version `v2.0.0` onwards)
 {{< /alert >}}
 
-{{< alert title="In Skaffold `v2` the primary difference between the helm renderer (`manifest.helm.*`) and the helm deployer `deploy.helm.*` is the use of `helm template` vs `helm install`" >}}
+{{< alert title="Note" >}}
+In Skaffold `v2` the primary difference between the helm renderer (`manifest.helm.*`) and the helm deployer (`deploy.helm.*`) is the use of `helm install` vs `helm install
```
Comment (ericzzzzzzz) [inline docs-v2/content/en/docs/pipeline-stages/renderers/helm.md:26]: should this be `helm template` vs `helm install` ?

**Comment by ericzzzzzzz** [comment]
Screen Shot 2022-11-18 at 12 17 24 PM
some texts are surrounded by plain Backticks, is this expected, or you were trying to do some formatting?

File: docs-v2/content/en/docs/pipeline-stages/deployers/helm.md (hunk 1 above)
Comment (aaron-prindle) [reply docs-v2/content/en/docs/pipeline-stages/deployers/helm.md]: thanks for catching this, this was not intentional.  Fixed now (to what you have above)

File: docs-v2/content/en/docs/pipeline-stages/renderers/helm.md (hunk 2 above)
Comment (aaron-prindle) [reply docs-v2/content/en/docs/pipeline-stages/renderers/helm.md]: thanks for catching this, this was not intentional. Fixed now (to what you have above)

**Comment by aaron-prindle** [comment]
> some texts are surrounded by plain Backticks, is this expected, or you were trying to do some formatting?

This is fixed now, this was due a backtick that was accidentally deleted which messed up the formatting.  Should be correct now

**Review by ericzzzzzzz — APPROVED** [review]

**Pull request merged**